packages/date-picker/src/DatePicker/components/Calendar/Calendar.spec.tsx

```diff
@@ -0,0 +1,47 @@
+// import { act, render, screen } from "@testing-library/react"
```

Comment:
Is this test obsolete now? Or going to be re-enabled later?

Reply:
Writing these now! Didn't mean for this to slip in.